test/Faker/Container/ContainerBuilderTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Test\Container;
+
+use Faker\Container\ContainerBuilder;
+use Faker\English\FileExtension;
+use Faker\Test\TestCase;
+use Psr\Container\ContainerInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+final class ContainerBuilderTest extends TestCase
+{
+    public function testBuildEmpty()
+    {
+        $builder = new ContainerBuilder();
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
```

Comment:
I assume that both `phpstan/phpstan` and `vimeo/psalm` will report this as a useless assertion, since `ContainerBuilder::build()` has a corresponding return type declaration.

Can we remove it?

Reply:
This test make sure that we don't explode if we try to build an empty container. The assertion is here because a test without assertion will give a warning.